source/uri/uris.ads

```diff
@@ -95,10 +95,7 @@ package URIs is
 
    package Conversions is
       function From_File (Full_Path : String) return URI_String
-        with Pre =>
-          Ada.Strings.Fixed.Equal_Case_Insensitive
-            (Ada.Directories.Full_Name (Full_Path),
-             Full_Path);
+        with Pre => GNAT.OS_Lib.Is_Absolute_Path (Full_Path);
```

Comment:
Good change but looks like unrelated to change "visibility" of URIs directory.

Reply:
Right, but I did make a note of it in the commit changelog.

Reply:
I'm not sure that implementation will still work with given precondition